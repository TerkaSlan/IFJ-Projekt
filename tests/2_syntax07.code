/*Should be error becasue run is returning int*/

class Main {

	static int run() {
		int a = 1;
		return a;		
	}
}