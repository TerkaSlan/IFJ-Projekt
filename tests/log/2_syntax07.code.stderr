ERROR: Semantic - In class Main there is no run method.
